CIP-0007/README.md

```diff
@@ -121,13 +107,28 @@ Pledge	Rewards	Benefit	Alt Rwd	Alt Bnft
 
 As you can see this gives meaningful pledge benefit rewards to pools pledging less than 1m ADA.
 
-## Implementations
+## Rationale: how does this CIP achieve its goals?
+
+Using the n-root curve pledge benefit shows a much more reasonable distribution of pledge related rewards which will encourage meaningful pledges from more pool operators thus making the network more secure against Sybil attacks.
+It also provides higher rewards for higher pledge without disproportionately rewarding a very few private pool operators who provide no additional security value to the network.
+This modification maintains the general principles of the current rewards equation and does not introduce any hard limits.
+It improves the incentives that were originally designed to make them more meaningful for the majority of pool operators.
 
-If someone will show me where the current maxPool reward equation is implemented in the code, I could produce an implementation of this change as a pull request.
+### Backward Compatibility
 
-## Copyright
+This proposal is backwards compatible with the current reward function by setting the curve_root parameter to 1.
 
-Copyright 2020 Shawn McMurdo
+## Path to Active
+
+### Acceptance Criteria
+
+Agreement by the Ledger team as defined in [CIP-0084](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0084) under _Expectations for ledger CIPs_ including "expert opinion" on changes to rewards & incentives.
+
+### Implementation Plan
+
+Author has offered to produce an implementation of this change as a pull request if shown where the current maxPool reward equation is implemented in the code.
```

Comment:
```suggestion
- [ ] Agreement by the Ledger team as defined in [CIP-0084](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0084) under _Expectations for ledger CIPs_ including "expert opinion" on changes to rewards & incentives.

- [ ] Author has offered to produce an implementation of this change as a pull request if shown where the current maxPool reward equation is implemented in the code.
```

Reply:
@KtorZ - will note that point for other RSS proposals which are up next for remodelling.

Reply:
... also @KtorZ didn't know the checkboxes were required, since I thought some remodels didn't impose it, but will keep going with the checkboxes from now on.